### PR TITLE
server, ui: move the debug landing page to the admin ui

### PR DESCRIPTION
**Pull request by BramGruneir**
- This adds a redirect from the old location.

screen shot 2017-08-10 at 16 05 00

### CODE REVIEW

**Comment by mrtracy** [comment]
We chatted about this in person; it's sound from a functional perspective, but I had the suggestion of refactoring it to appear more like JSX code, which I think can make it significantly easier to understand.  Here was the brief example code we worked out to illustrate the concept:

```typescript
function renderDebugTable(props: {heading: string, children?: React.ReactNode}) {
  return (
    <div>
      <h3>{heading}</h3>
      <table className="debug-table">
        <tbody>
          {props.children}
        </tbody>
      </table>
    </div>
  );
}

<DebugTable title="Reports">
  <DebugSection title="NodeDiagnostic">
    <DebugLink name="All Nodes" url="#/reports/nodes" />
     <DebugLink name="All Nodes" url="#/reports/nodes" />
     <DebugLink name="All Nodes" url="#/reports/nodes" />
   </DebugSection>
 </DebugTable>
```

---

Review status: 0 of 5 files reviewed at latest revision, all discussions resolved, all commit checks successful.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/cockroachdb/cockroach/17635#-:-KrWzYNcw76cS06ew8bK:b-brc1r9)*

**Comment by BramGruneir** [comment]
Done!  PTAL.

---

Review status: 0 of 5 files reviewed at latest revision, all discussions resolved.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/cockroachdb/cockroach/17635#-:-KrXF2uy-rbr_2h1BpFf:b-kthq2j)*

**Comment by mrtracy** [comment]
:lgtm:

---

Review status: 0 of 5 files reviewed at latest revision, all discussions resolved, all commit checks successful.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/cockroachdb/cockroach/17635#-:-KrXL68Kzh8ZtRVZMArE:bnfp4nl)*

File: pkg/ui/src/views/reports/containers/debug/index.tsx [hunk 1]
```diff
@@ -0,0 +1,202 @@
...
+            name="Latency filtered by locality (regex)"
+            url="#/reports/network?locality=region=us-east"
+            note="#/reports/network?locality=[regex]"
+          />
+        </DebugTableRow>
+        <DebugTableRow title="Security">
+          <DebugTableLink name="Certificates on this node" url="#/reports/certificates/local" />
+          <DebugTableLink
+            name="Certificates on a specific node"
+            url="#/reports/network?node_ids=1,2"
```
Comment (mberhault) [inline pkg/ui/src/views/reports/containers/debug/index.tsx]: This got mixed up a bit: title says `certificates on this node`, but link is for `network?node_ids=1,2`

File: pkg/ui/src/views/reports/containers/debug/index.tsx [hunk 2]
```diff
@@ -0,0 +1,202 @@
...
+            note="#/reports/network?node_ids=[node_id{,node_id...}]"
+          />
+          <DebugTableLink
```
Comment (mberhault) [inline pkg/ui/src/views/reports/containers/debug/index.tsx]: ditto here: name is `latency filtered by locality`, but link is `certificates/1`

**Comment by BramGruneir** [comment]
Yep.  Got a fix for this incoming today.  It got messed up in my
refactoring of how the page was written.

Got a few fixes for these pages, just bundling them together due to
embedded.go being a pain to work with.

**Pull request merged**